components/core/JsInterop/interop.ts

```diff
@@ -125,7 +125,8 @@ export function uploadFile(element, index, data, headers, fileId, url, name, ins
   const req = new XMLHttpRequest()
   req.onreadystatechange = function () {
     if (req.readyState === 4) {
-      if (req.status != 200) {
+      // #1655 Any 2xx response code is okay
+      if (req.status < 200 || req.status > 299) {
```

Comment:
It should be less than 300.

Reply:
You mean to say that 1xx is fine too? Because the code i modified would trigger error method only if status is less than 200 or greater than 299.

Reply:
Ok, you are right.